### ISSUE TITLE
[Feature] AnimalSubType conversion

### ISSUE THREAD

**Issue by inconspicuously007** [tn4799/AnimalInputExtension#3]
Hey,

aktuell sind die Tier-Filltypes an den Storage der Produktion gebunden, was ja auch genau so funktioniert.
Jedoch ist das nicht unbedingt der korrekte Platz, diese wäre im Trigger besser aufgehoben, denke ich. Denn so wäre eine mögliche Erweiterung realisierbar, die einer FillTypeConversion, wie sie bei UnloadTriggern möglich ist, nahe kommt. So könnte man den jeweiligen AnimalSubType in ein für die Produktion generell nutzbaren FillType "umwandeln".
Zum Beispiel ist Rindfleisch aus einer Kuh, da kann es dann egal sein welcher Rasse diese Kuh hat. Ähnlich verhält es sich bei Schweinen.

Dies soll lediglich als Anregung dienen, das Script ist ohne Frage gut und bringt viel mehr Möglichkeiten.

MfG

**Comment by tn4799** [comment]
Die fillTypes müssen irgendwo gelagert werden, daher der storage, vor allem da giants ihre produktionen da drauf zugreifen. 
Aber kann man sich mal überlegen, eine Conversion einzubauen.

**Comment by inconspicuously007** [comment]
Ja, korrekt, aber mit der Conversion muss im Storage der Produktion nur noch der Ziel-FillType der Conversion enthalten sein.
Des Weiteren kann dies die Produktion verschlanken, da man Produktionslinien spart. Es ist von Giants ja angedacht, dass jede Linie einen Input-FillType hat und keine FillTypeCategories unterstützt werden.
